airflow/logging/airflow_logging_config.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
```

Comment:
I prefer `airflow.utils.log.config.py` or just have it in `airflow.utils.log.py`. Please don't refer to airflow twice and logging twice.